FE/src/App.jsx

```diff
@@ -1,5 +1,33 @@
+import { useState } from 'react'
+import { Routes, Route} from 'react-router-dom'
+import NavBar from './components/NavBar'
+import MainPage from './components/MainPage'
+import Recipes from './components/Recipes'
+import SingleRecipe from './components/SingleRecipe'
+import NewRecipe from './components/NewRecipe'
+import EditRecipe from './components/EditRecipe'
+import Login from './components/Login'
+import Account from './components/Account'
+import Admin from './components/Admin'
+
 function App() {
-  return <>Hello Lab Rats</>;
+  const [token, setToken] = useState(null)
+
+  return (
+    <>Hello Lab Rats
```

Comment:
Might should wrap this in an header or p tag. Not sure if having unwrapped text will negatively affect anything